101-application-gateway-rewrite/README.md

```diff
@@ -0,0 +1,107 @@
+2.	You will then be required to attach the rewrite rule set with a routing rule. Once created, this rewrite configuration is attached to the source listener via the routing rule. When using a basic routing rule, the header rewrite configuration is associated with a source listener and is a global header rewrite. When a path-based routing rule is used, the header rewrite configuration is defined on the URL path map. So, it only applies to the specific path area of a site.
+
+You can create multiple http header rewrite rule sets and each rewrite rule set can be applied to multiple listeners. However, you can apply only one http rewrite rule set to a specific listener.
+
+| Property | Required | Description |
+|---|---|---|
+| rewriteRuleSets[] | No | List of rewrite rule sets. Each rewriteRuleSet contains a set of rules which have a sequence associated with them. |
+| rewriteRuleSets[].rules[] | Yes | Set of rules which are grouped together which can be associated to one or more request routing rule(s). |
+| rules[].actionSet | Yes | Set of actions to take when all the conditions are met. |
+| rules[].requestHeaderConfiguration[] | No | List of request header actions to take. Either sets or deletes the request headers based on the values of headers or server variables. |
+| requestHeaderConfiguration[].headerName | Yes | Name of the header to set. |
+| requestHeaderConfiguration[].headerValue | Yes | Value to set for the header. It can be a constant string or a format string. If the value is set to an empty string, the header will be removed from the HTTP packet.
+
+### Sample Rewrite Configuration
+```
+{
+    "rewriteRuleSet": [
+        {
+        "name": "RewriteRuleSet1",
+        "properties": {
+            "rewriteRule": [
+            {
+                "name": "RWRule1",
+                "actionSet": {
+                "requestHeaderConfiguration": [
+                    {
+                    "headerName": "X-Forwarded-For",
+                    "headerValue": "source IP"
+                    },
+                    {
+                    "headerName": "Ciphers-Used",
+                    "headerValue": "{var_ssl_cipher}"
+                    }
+                ],
+                "responseHeaderConfiguration": [
+                    {
+                    "headerName": "Strict-Transport-Security",
+                    "headerValue": "max-age=31536000"
+                    }
+                ]
+                }
+            }
+            ]
+        },
+        "type": "Microsoft.Network/applicationGateways/rewriteRuleSet"
+        }
+    ]
+}
+```
+
+
+### Sample request routing rule configuration with rewrite rule set
+```
+{
+    "requestRoutingRules": [
+        {
+            "Name": "rule1",
+            "properties": {
+                "RuleType": "Basic",
+                "httpListener": {
+                "id": "[concat(variables('applicationGatewayID'), '/httpListeners/appGatewayHttpListener')]"
```

Comment:
remove concat and use resourceId for all the resourceIds

Reply:
Removed the concat and kept just the id ("/httpListeners/appGatewayHttpListener")

Reply:
Done.

Reply:
I don't see the change in the PR...

Reply:
updated now.